internal/jimm/bootstrap/bootstrap.go

```diff
@@ -147,14 +147,58 @@ func (b *bootstrapManager) StopBootstrap(ctx context.Context, user *openfga.User
 		return errors.E(op, "job ID cannot be nil")
 	}
 
-	err := b.jobtracker.StopJob(ctx, jobId)
+	err := b.tracker.StopJob(ctx, jobId)
 	if err != nil {
 		return errors.E(op, "failed to stop job", err)
 	}
 
 	return nil
 }
 
+// StartBootstrap starts a bootstrap job with the provided parameters.
+func (b *bootstrapManager) StartBootstrap(ctx context.Context, user *openfga.User, params BootstrapParams) (string, error) {
+	const op = errors.Op("jimm.StartBootstrap")
+
+	if err := params.validate(); err != nil {
+		return "", errors.E(op, fmt.Errorf("invalid bootstrap parameters: %v", err))
+	}
+
+	temp, err := os.MkdirTemp("", "juju-data-dir")
+	if err != nil {
+		return "", errors.E(op, fmt.Errorf("failed to create temporary directory for Juju data: %w", err))
+	}
+
+	jobId, err := b.tracker.Run(
+		ctx,
+		bootstrapJobType,
+		b.BootstrapJob(
+			JobParams{
+				// Runner args.
+				JujuDataDir: temp,
+				// Binary args.
+				CLIVersion: params.CLIVersion,
+				// User defined command arguments
+				CloudNameAndRegion: params.CloudNameAndRegion,
+				ControllerName:     params.ControllerName,
+				AgentVersion:       params.AgentVersion,
+				BootstrapTimeout:   params.BootstrapTimeout,
+				CloudCred:          params.CloudCred,
+				PersonalCloud:      params.PersonalCloud,
+				// JIMM Provided command arguments (i.e., ones that must be set by JIMM when bootstrapping).
+				LoginTokenRefreshURL: b.jimmWellknownJWKSEndpoint, // TODO: Set the correct login token refresh URL
```

Comment:
Feels like this could be split up nicely in 2, one struct for bootstrapArgs and another for commandArgs